ansible/playbook/tasks/roles/initialization/tasks/main.yml

```diff
@@ -1,5 +1,9 @@
 ---
 # This role contains tasks executed during initialization of PMM Server
+- name: Update repository settings
+  command: yum-config-manager --setopt=epel.timeout=1 --save
+  changed_when: True
```

Comment:
Is it `changed_when` or `ignore_errors`, that we need to use here?

Reply:
`ansible-lint` was throwing following error:

> [301] Commands should not change things if nothing needs doing

When you run command or shell, they always set **changed** to **True**. This is because Ansible has no mechanism for understanding whether your command changed anything or not. Some commands are genuinely read only (e.g. git status) and others have side effects. And the command we used always change the repo conf, thats why I added `changed_when: True`.